m/mangohud/stone.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : mangohud
+version     : 0.7.2
+release     : 1
+homepage    : https://github.com/flightlessmango/MangoHud/releases/download/v0.7.2
+upstreams   :
+    - https://github.com/flightlessmango/MangoHud/releases/download/v0.7.2/MangoHud-v0.7.2-Source.tar.xz : 114ad3ea87b1db7358816c7b8e7843aaee356ff048b9e15d6fff02d89414841b
+summary     : A Vulkan and OpenGL overlay for monitoring FPS, temperatures, CPU/GPU load and more
+description : |
+    A Vulkan and OpenGL overlay for monitoring FPS, temperatures, CPU/GPU load and more.
+license     : MIT
+toolchain   : gnu
```

Comment:
Why? Is there a build failure? What is it? Such information should be in the recipe as a comment.